Gopkg.toml

```diff
@@ -41,20 +41,20 @@ required = [
   version = "kubernetes-1.13.1"
 
 [[constraint]]
-  name="sigs.k8s.io/controller-tools"
-  version="v0.1.1"
+  name = "sigs.k8s.io/controller-tools"
+  version = "v0.1.9"
 
 [[constraint]]
-  name="github.com/softlayer/softlayer-go"
-  branch="master"
+  name = "github.com/softlayer/softlayer-go"
+  revision = "7c592eb"
```

Comment:
+1 to use revision here, this can make sure us to a stable release or continue use this revision when release the ibmcloud provider.

@xunpan @jichenjc for comments.

Reply:
I think it is fine for softlayer-go because it never has an official release. A snapshot version is better to avoid updating vendors frequently without expectation.

But for `cluster-api`, I prefer to use a release version unless there has strong reason to use a specific snapshot.

Reply:
agree, we can bump the version if need

for cluster-api, we are using 0.1-release branch now, so we should be fine